pkg/v1/tkg/web/package.json

```diff
@@ -17,7 +17,8 @@
     "aws-e2e": "cd e2e && npm run aws-e2e",
     "vsphere-e2e": "cd e2e && npm run vsphere-e2e",
     "azure-e2e": "cd e2e && npm run azure-e2e",
-    "build-index": "node src/app/shared/contextual-help/build-script/build.js"
+    "build-index": "node src/app/shared/contextual-help/build-script/build.js",
+    "convertMd": "node src/app/shared/contextual-help/build-script/convert.js && npm run build-index"
```

Comment:
one nitpick: rename "convertMd" to "convert-md-to-html"

Reply:
renamed